airflow/utils/log/cloudwatch_task_handler.py

```diff
@@ -0,0 +1,114 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import watchtower
+from cached_property import cached_property
+
+from airflow.configuration import conf
+from airflow.utils.log.file_task_handler import FileTaskHandler
+from airflow.utils.log.logging_mixin import LoggingMixin
+
+
+class CloudwatchTaskHandler(FileTaskHandler, LoggingMixin):
+    """
+    CloudwatchTaskHandler is a python log handler that handles and reads task instance logs.
+
+    It extends airflow FileTaskHandler and uploads to and reads from Cloudwatch.
+
+    :param base_log_folder: base folder to store logs locally
+    :type base_log_folder: str
+    :param log_group_arn: ARN of the Cloudwatch log group for remote log storage
+    :type log_group_arn: str
+    :param filename_template: template for file name (local storage) or log stream name (remote)
+    :type filename_template: str
+    """
+    def __init__(self, base_log_folder, log_group_arn, filename_template):
+        super().__init__(base_log_folder, filename_template)
+        split_arn = log_group_arn.split(':')
```

Comment:
Could you add an example ARN here in a comment? Makes it easier to tell/sanity check what the 4th and 7th parts are.